gapic/src/main/com/google/gapid/perfetto/views/GpuQueuePanel.java

```diff
@@ -127,6 +128,11 @@ public void renderTrack(RenderContext ctx, Repainter repainter, double w, double
     });
   }
 
+  private static String getTitle(String title, ArgSet args) {
```

Comment:
Bad function name. It's not getting the title, it's constructing it. Can you change it with createTitle or constructTitle?

Reply:
done. 🙄